app/export_test.go

```diff
@@ -214,3 +214,162 @@ func TestExportAllUsers(t *testing.T) {
 	assert.Equal(t, len(deletedUsers1), len(deletedUsers2))
 	assert.ElementsMatch(t, deletedUsers1, deletedUsers2)
 }
+
+func TestExportDMChannel(t *testing.T) {
+	th1 := Setup(t).InitBasic()
+	defer th1.TearDown()
+
+	// DM Channel
+	th1.CreateDmChannel(th1.BasicUser2)
+
+	var b bytes.Buffer
+	err := th1.App.BulkExport(&b, "somefile", "somePath", "someDir")
+	require.Nil(t, err)
+
+	result := <-th1.App.Srv.Store.Channel().GetAllDirectChannelsForExportAfter(1000, "00000000")
+	channels := result.Data.([]*model.DirectChannelForExport)
+	assert.Equal(t, 1, len(channels))
+
+	th2 := Setup(t)
```

Comment:
OK. There was some bias from other tests in which I understood they'd create a different db and hence would serve as clean slate for an import.
The idea is indeed to check the effect on a new database so I think an explicit teardown would make sense after the export (instead of deferring it).